src/lib/projects/YearlyEvents.svelte

```diff
@@ -19,18 +47,26 @@
 			<label class="ml-4" for="only-members">Nur Mitglieder</label>
 		</div>
 	</div>
-	<section class="mt-12">
-		<img src={EventImage} alt="" class="mt-12 rounded-t-2xl" />
-		<div class="p-6 rounded-b-2xl bg-soft-yellow flex">
-			<div class="flex flex-col justify-center items-center">
-				<h1 class="text-5xl font-serif font-bold">24</h1>
-				<h1>August</h1>
+	{#each events as event}
+		<section class="mt-12">
```

Comment:
@Han2-Ro This works as well and I think it would make sense to wrap all three events inside the section tag. 

Then we could make the events become an unordered list with `<ul>`. This will make the accessibility better, as screen readers and other tools will know that the events "belong together".

What I mean is something like this:

```html
<section>
  <ul>
     <li>First Event</li>
     <li>Second Event</li>
     ...
  </ul>
</section>
```

See also [the usage notes on `<section>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#usage_notes)

Reply:
I removed the section tag and replaced it with <ul> outside of the each loop und <li> inside the loop.